### DISCUSSION TITLE
How can i use 2 supabase projects, from same organization in one flutter app? Is it possible to have so

### DISCUSSION THREAD

**Discussion by ijaspreetsinghh** [supabase/supabase#5781] [Questions] [answered]
I have one main supabase project for my flutter app and looking for a way to connect another supabase project to the same flutter app,

**Comment by w3b6x9** [comment, marked as answer]
@ijaspreetsinghh if i'm not mistaken, i think you can just create another client connected to the second supabase project. have you tried that?

**Reply by ijaspreetsinghh** [reply]
that worked man, thanks.

**Reply by jstdk** [reply]
@ijaspreetsinghh, @w3b6x9  it works, but the only drawback is that this message appears in recent Supabase versions

![image](https://github.com/supabase/supabase/assets/115603308/ccb5705e-85e8-41da-b20b-396f03511f29)

**Reply by w3b6x9** [reply]
@kangmingtay would you know what's going on here?

**Comment by jstdk** [comment]
@w3b6x9, how would that code look like? 

Below code does not work
![image](https://github.com/supabase/supabase/assets/115603308/6e68a400-3620-48ed-bae3-cf639c2682a1)

main.dart
```
await Supabase.initialize(
      url: dotenv.env['SUPABASE_URL_1']!,
      anonKey: dotenv.env['SUPABASE_KEY_1']!);
  await Supabase.initialize(
      url: dotenv.env['SUPABASE_URL_2']!,
      anonKey: dotenv.env['SUPABASE_KEY_2']!); 
```

somewhere.dart
`final supabase = Supabase.instance.client;
`

Thanks

**Reply by w3b6x9** [reply]
@dshukertjr can you take a look at this?

**Reply by dshukertjr** [reply]
@jstdk 
You can create your second Supabase instance manually like this:

```dart
final supabase = SupabaseClient('supabase_url', 'supabase_anon_key');
```

**Reply by jstdk** [reply]
Thanks @dshukertjr. That works indeed, That leaves the issue I left above about the warning of 2 GoTrue instances in parallel, happens both in Javascript and Flutter

**Reply by dshukertjr** [reply]
@jstdk You only need to worry about the warning if you are accessing the same Supabase instance from two difference Gotrue instance. In this case, you are accessing two different Supabase instance from two differnet Gotrue instances, so it should be fine.

**Reply by jstdk** [reply]
Thanks @dshukertjr. Hope in future the warning goes away in case there are 2 clients to 2 instances (instead of 2 clients to 1 instance), as it keeps the logs clean :)
